definitions/contested/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -3442,7 +3442,7 @@
     "CaseEventID": "FR_amendApplication",
     "CaseFieldID": "applicantSolicitorEmail",
     "PageFieldDisplayOrder": 9,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "OPTIONAL",
```

Comment:
This field is mandatory during paper case creation

Reply:
Ok I've corrected this. It does appear when Applicant is not represented - which didn't look right - could this be a bug?
Should there be this condition like as in FR_newPaperCase?:         "FieldShowCondition": "applicantRepresented=\"Yes\""

Reply:
It doesn't look like there's even an option to select whether the applicant is represented for the amend case event, which would be why there's no field show condition